--- ssh.py.old	2016-03-13 18:11:13.246135753 +0100
+++ ssh.py	2016-03-13 18:11:31.798898481 +0100
@@ -569,10 +569,11 @@
         # python interactive-mode but the modules are not compatible with the
         # interactive-mode ("unexpected indent" mainly because of empty lines)
 
-        if in_data:
-            cmd = self._build_command('ssh', self.host, cmd)
-        else:
-            cmd = self._build_command('ssh', '-tt', self.host, cmd)
+        #if in_data:
+        #    cmd = self._build_command('ssh', self.host, cmd)
+        #else:
+        #    cmd = self._build_command('ssh', '-tt', self.host, cmd)
+        cmd = self._build_command('ssh', self.host, cmd)
 
         (returncode, stdout, stderr) = self._run(cmd, in_data, sudoable=sudoable)
 
